doc/rm/comportability_specification/index.md

```diff
@@ -309,6 +309,7 @@ The following standardised countermeasures are defined:
 | LOCAL_ESC      | A local escalation event is triggered when an attack is detected |
 | GLOBAL_ESC     | A global escalation event is triggered when an attack is detected |
 | UNPREDICTABLE  | Behaviour is unpredictable to frustrate repeatable FI attacks |
+| TERMINAL       | The asset goes into a terminal statet that no longer responds to any stimulus |
```

Comment:
```suggestion
| TERMINAL       | The asset goes into a terminal state that no longer responds to any stimulus |
```

Reply:
@msfschaffner does this make sense to you?
Basically we realized during v2s we need to differentiate between local escalate in pwrmgr and this kind of terminal kill. Because the local escalate technically results in a reset whereas terminal does not.